content/docs/ethernet-v0.5/environment.md

```diff
@@ -0,0 +1,109 @@
++++
+Description = ""
+date = "2017-10-14T13:00:00+00:00"
+title = "Prepare the environment"
+parent = "/docs/ethernet-v0.5/"
+prev = "/docs/ethernet-v0.5/softwaremethodology/"
+next = "/docs/ethernet-v0.5/installtools/"
+showdisqus = true
+
++++
+
+lowRISC tools and source code are robust between versions of Linux, but it has been
+found that
+Vivado (the Xilinx FPGA implementation suite) [is particular about the O/S](https://www.xilinx.com/support/answers/54242.html)
+
+All builds were completed with Vivado 2015.4. If a newer version is attempted there could be script incompatibilities to fix and/or incompatible upgrades to internal Xilinx IP. We have no evidence that newer versions do not work, however IP changes to support newer chip families can cause obscure error messages which are offputting for the first-time user.
+
+For this version the recommended O/S 64-bit Ubuntu (14.04 LTS). However, this is not stable on newer Intel chip sets, where 16.04 LTS should be used instead. If an unstable choice is made, the most likely result is LD\_LIBRARY\_PATH conflicts and/or synthesis crashes. It should be safe to unset LD\_LIBRARY\_PATH in your shell. This will prevent conflicts with the obsolete libstc++ build-in to Vivado.
```

Comment:
Is there a link to the bug Ubuntu bug described here? If Ubuntu 16.04 is known to work, perhaps we should just say that this has been tested with Ubuntu 14.04 and 16.04?